hub/values.yaml

```diff
@@ -32,6 +32,20 @@ jupyterhub:
     service:
       type: ClusterIP
     chp:
+      # testing out a potential fix for the chp leaking ports
+      # https://github.com/jupyterhub/configurable-http-proxy/issues/434
+      # https://github.com/jupyterhub/configurable-http-proxy/issues/434#issuecomment-1924261459
+      # https://github.com/http-party/node-http-proxy/pull/1559
+      # https://github.com/Jimbly/http-proxy-node16/commit/56283e33edfc7aad8c2605dd493da8a196b4371d
+      # https://github.com/consideRatio/configurable-http-proxy/commits/main/
+      #
+      image:
+        tag: 4.6.1-fork
+      extraCommandLineFlags:
+        # set the timeout and proxyTimeout to 24 hours (arg is in milliseconds)
+        # https://github.com/http-party/node-http-proxy?tab=readme-ov-file#options
+        - "--timeout 86400000"
+        - "--proxyTimeout 86400000"
```

Comment:
```suggestion
        - "--proxyTimeout=86400000"
```

Reply:
Without this i expect its like a string with a space in it will be considered an invalid flag

Reply:
@consideRatio For what it is worth, an example in the docs has a space.

https://z2jh.jupyter.org/en/stable/resources/reference.html#proxy-chp-extracommandlineflags
```yaml
proxy:
  chp:
    extraCommandLineFlags:
      - "--auto-rewrite"
      - "--custom-header {{ .Values.myCustomStuff }}"
```

Reply:
intersting.  i'll leave the `=` sign there.